### DISCUSSION TITLE
Whatever happened to i3wm?

### DISCUSSION THREAD

**Discussion by jenstroeger** [Homebrew/discussions#3627] [Casks] [answered]
I remember installing [i3wm](https://i3wm.org/) with Brew because I wasn’t able to build it myself (due to missing libraries). That was a few years ago and I haven’t touched it since.

Alas, this week I installed [protolint](https://github.com/yoheimuta/protolint) using Brew which updated most of the rest… and broke i3wm.

Which seems to have disappeared entirely? What happened?

**Comment by SMillerDev** [comment, marked as answer]
There is no record of i3wm ever existing in official repositories as far as I could tell. Maybe you installed it based on brew libraries instead?

**Comment by jenstroeger** [comment]
I remember issues with [xkbcommon](https://xkbcommon.org/) which didn’t exist back then on MacPorts, so I opened a ([ticket](https://trac.macports.org/ticket/52596)). On Brew, that library existed and perhaps I installed that one… 🤔 It’s been a while.

How would I go about requesting an `i3wm` package? (Considering MacPorts’s package [is broken](https://trac.macports.org/ticket/65811)). Or should I build and install `i3wm` from source locally, because there may not be interest on the Brew/MacPorts side?

**Reply by SMillerDev** [reply]
https://docs.brew.sh/Adding-Software-to-Homebrew explains how to add software to homebrew.

**Comment by p-linnane** [comment]
@jenstroeger Take a look at [Adding Software To Homebrew](https://docs.brew.sh/Adding-Software-to-Homebrew).
